src/pages/registration/new_registration/StartEmptyRegistration.tsx

```diff
@@ -27,7 +27,7 @@ export const StartEmptyRegistration = ({ onChange }: Pick<StartRegistrationAccor
       dispatch(setNotification({ message: t('feedback.error.create_registration'), variant: 'error' }));
       setIsLoading(false);
     } else if (isSuccessStatus(createRegistrationResponse.status)) {
-      history.push(getRegistrationWizardPath(createRegistrationResponse.data.identifier), { highestValidatedTab: -1 });
+      history.push(getRegistrationWizardLink(createRegistrationResponse.data.identifier, -1));
```

Comment:
Umulig å vite hva `-1` betyr her, uten å hoppe videre inn i fila hvor funksjonen er, og lese koden der. Det er
et argument for at andre parameter her kunne vært et objekt, sånn at det fortsatt blir `{ highestValidatedTab: -1 }` i stedet, og man skjønner med en gang hva `-1` representerer